### PR TITLE
53 setup step by step

### LINKED ISSUE #53: Setup step-by-step
- Create step by step registration page.
- Implement first step: 

> The first step is to fill in:
> Name, Surname, username can be edited (by default it will be truncated mail), Date of birth, Phone number, and gender

### PR DESCRIPTION
**Pull request by maryanchuk04**

### CODE REVIEW

File: StudCity.Frontend/src/UI/DatePicker/index.jsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
+/* eslint-disable react/display-name */
+import React, { useRef } from "react";
+import ReactDatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
```
Comment (Serhiy-dotcom) [inline StudCity.Frontend/src/UI/DatePicker/index.jsx]: I would suggest in the future put styling imports at the end of import chain

File: StudCity.Frontend/src/pages/RegistrationComplete/index.jsx [hunk 2]
```diff
@@ -0,0 +1,94 @@
+import React, { useState, useEffect } from "react";
+import Stepper from "../../components/Stepper/Stepper";
+import UserInformation from "./steps/UserInformation";
+import { useSelector, useDispatch } from "react-redux";
```
Comment (Serhiy-dotcom) [inline StudCity.Frontend/src/pages/RegistrationComplete/index.jsx:2]: same here but putting import of libraries at the top

File: StudCity.Frontend/src/pages/RegistrationComplete/index.jsx [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+	const [valid, setValid] = useState(false);
+	const [state, setState] = useState(getRegisterCompleteDefautlData(defaultData));
+
+	useEffect(() => {
+		setValid(stepValidator());
+	}, [state])
+
+	const handleNext = () => {
+		dispatch(changeRegisterCompleteState(state));
+		if (activeStep === 5) {
```
Comment (Serhiy-dotcom) [inline StudCity.Frontend/src/pages/RegistrationComplete/index.jsx]: maybe it's a good idea to put 5 into constants folder as I'm seeing a lot of usage of it, consider replacing 5 with a constant in all places where it's used

**Review by Serhiy-dotcom — CHANGES_REQUESTED** [review]
Please make small fixes, overall look good👍

**Review by Serhiy-dotcom — APPROVED** [review]
Well done!

**Review by Vova-Romaniuk — APPROVED** [review]
i`m hater class component, but it`s good PR

**Pull request merged**